File: public/usage-examples/animations/has_animation_script-1-example.py
from splashkit import *

def main():
    load_animation_script_named('WalkingScript', 'Resources/animations/kermit.txt')
    present = has_animation_script('WalkFront')

    if present:
        write_line('Has animation script: true')
    else:
        write_line('Has animation script: false')

    free_animation_script_with_name('WalkingScript')

if __name__ == '__main__':
    main()

File: public/usage-examples/animations/has_animation_script-1-example.cpp
#include "splashkit.h"

int main()
{
    // Load script so frames are present
    load_animation_script_named("WalkingScript", "Resources/animations/kermit.txt");

    // Check globally for the frames for the named animation
    bool present = has_animation_script("WalkFront");
    write_line(present ? "Has animation script: true" : "Has animation script: false");

    // cleanup via name
    free_animation_script_with_name("WalkingScript");
    return 0;
}
